modules/ddl-and-loading/pages/managing-loading-job.adoc

```diff
@@ -224,8 +224,34 @@ The above loading job and data generate the following reports:
   +--------------------------------------------------------------------------------------+
 [WARNING] bad data in m1 (replica 1) /home/tigergraph/movie.dat: 1 line(s) do not have enough number of tokens.
 [WARNING] bad data in m1 (replica 1) /home/tigergraph/movie.dat:Movie: 1 object(s) have invalid attributes.
+Sampling error data can be viewed by executing the 'SHOW LOADING ERROR <job_id>'.
 ----
 
+As mentioned in the report, The command `SHOW LOADING ERROR <job_id>` shows the error data sampled during data loading.
```

Comment:
```suggestion
== `SHOW LOADING ERROR`

Beginning with TigerGraph 4.1, the loader will log a sample of the data lines that it could not load.
The command `SHOW LOADING ERROR <job_id>` presents a table of the sample of malformed data lines.
Each row reports not only a data line and its location, but also its error type.  This information helps users to quickly identify and understand errors, leading to quicker resolution of data format problems.

```